---
layout: page
title: About me
subtitle: Servus! (that means "hi" in bavarian)
---

My name is Stefan Peidli. I'm a PhD student at the [Blüthgen Lab](https://www.sys-bio.net/) in Berlin, Germany.

### My Current Research
- Perturbation biology (drugs, crispr, ...)
- Cancer (specifically colon and lung)
- SARS-CoV2

### Research Interests
- All things single cell
- Manifold Learning
- Deep Learning
- Immunology
- Fast Evolution (e.g. of Cancer or Viruses)

### My story
- PhD in Computational Biology in Berlin (ongoing)
- Started working at Theislab, found out I like computational biology
- Master in Mathematics at [TU Munich](https://www.tum.de/en/)
- Bachelor in Mathematics with Minor in Physics at [TU Munich](https://www.tum.de/en/)

### Favorite Papers
- Haghverdi, L., Büttner, M., Wolf, F. A., Buettner, F., & Theis, F. J. (2016). Diffusion pseudotime robustly reconstructs lineage branching. Nature methods, 13(10), 845-848.
- Love, M. I., Huber, W., & Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome biology, 15(12), 1-21.
- Lipton, Z. C. (2018). The mythos of model interpretability: In machine learning, the concept of interpretability is both important and slippery. Queue, 16(3), 31-57.
- Hinton, G. E. (2002). Training products of experts by minimizing contrastive divergence. Neural computation, 14(8), 1771-1800.
- Luecken, M. D., & Theis, F. J. (2019). Current best practices in single‐cell RNA‐seq analysis: a tutorial. Molecular systems biology, 15(6), e8746.
- Lopez, R., Regier, J., Cole, M. B., Jordan, M. I., & Yosef, N. (2018). Deep generative modeling for single-cell transcriptomics. Nature methods, 15(12), 1053-1058.
- Schiebinger, G., Shu, J., Tabaka, M., Cleary, B., Subramanian, V., Solomon, A., ... & Lander, E. S. (2019). Optimal-transport analysis of single-cell gene expression identifies developmental trajectories in reprogramming. Cell, 176(4), 928-943.
